### PR TITLE
Seven Bars to Code at in Brooklyn

### PR DESCRIPTION
**Pull request by ptsteadman**

### CODE REVIEW

File: source/_posts/bars-to-code-at.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+Finally, after a good coding session, I often find myself in a sort of distant,
+aphasic state. Everything seems flat and it's hard to think of anything to say.
+Being at a bar gives me a chance to turn off my computer, find someone to talk
+to, and reintegrate.
+
+Of course, not all bars are good places to code. Some bars are problematic for
+social reasons, and others are problematic for practical reasons. Here's a list
+of bars that Computer Lab members have found suitable for computer use.
+
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: Consider removing first bits of each paragraph (first,second,third,finally,of course)

File: source/_posts/bars-to-code-at.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+category: Notes
+lede: In my opinion, bars are a better place to code than coffee shops.
+author: Patrick Steadman
+published: true
+image: "/images/coding-at-bar.jpg"
+---
+
+<center>![bars to code at](/images/coding-at-bar.jpg)</center>
+
+__In my opinion, bars are a better place to code than coffee shops.__
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: Consider removing in my opinion

File: source/_posts/bars-to-code-at.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+### Sugarburg
+Williamsburg [Map](https://goo.gl/maps/A2iQNHNmyTs)
+
+[sugarburg.com](http://sugarburg.com)
+
+Sugarburg features great natural light, unconventional seating, above-par bar
+food, and good service. The vertically slatted windows make this a good place to
+enjoy the weather during the spring and summer. The bartenders encourage people
+to use the bar as a workspace during the day, making Sugarburg a good place to
+work or meet regularly. The bar is large and Bourbun-oriented, making Sugarburg
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: s/Bourbun/Bourbon

File: source/_posts/bars-to-code-at.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+### Boobie Trap
+Bushwick [Map](https://goo.gl/maps/Q3CUv7sTRVE2)
+
+[boobietrapbrooklyn.com](http://boobietrapbrooklyn.com)
+
+[Robert Jensen](http://you.could.use.some.r1b.solutions/) says that Boobie Trap
+is actually a pretty horrible place to code (it's very loud and the tables are
+sticky), but it's one of the few places along the L in Bushwick with good
+internet.  He goes there to push docker containers and backups.
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: Mention that they have a dedicated charging station for phones & a power strip next to that.

File: source/_posts/bars-to-code-at.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+lighting and acoustics. The menu is varied and extensive. But 61 Local opens
+early, so outlets can be hard to find, and closes around midnight (relatively
+early).
+
+### Mother's
+Williamsburg [Map](https://goo.gl/maps/So1UNBjJpio)
+
+[mothersbrooklyn.com](https://mothersbrooklyn.com)
+
+According to Kelsey Hunter, the ideal Mother's coding session starts with a
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: Does Kelsey have a homepage to link to?

File: source/_posts/bars-to-code-at.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+$12 lunch special lasts until 5PM.  They have TVs with movies so you have
+something to look at while absorbing the sound of other's social interactions.
+
+### The Roof
+Gowanus [Map](https://goo.gl/maps/SwRfPpz7AbC2)
+
+[wholefoodsmarket.com/stores/thirdand3rd](http://www.wholefoodsmarket.com/stores/thirdand3rd)
+
+The Roof is the bar on the roof of the Gowanus Whole Foods, a favorite coding
+spot of influential Twitter user [Sbarro
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: lmao

File: source/_posts/bars-to-code-at.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+### Flowers For All Occasions
+Bedford-Stuyvesant [Map](https://goo.gl/maps/2KfMCU1C95x)
+
+[flowersforalloccasions.org](http://flowersforalloccasions.org)
+
+Flowers For All Occasions doesn't serve liquor, which can be a feature if you're
+looking for a place without lots of very drunk people.  In general the vibe
+tends to reflect the place's name.  It's a pleasant place for conversation,
+quiet group activities, or thoughtful work. I was introduced to Flowers For All
+Occasions by Computer Lab Writer in Residence Theo Thimo, who often uses it as a
```
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: Link to theo twitter?

**Comment by ptsteadman** [comment]
Merging in order to see if the image meta works.

**Pull request merged**